src/CompositeContainer.php

```diff
@@ -18,12 +19,15 @@ class CompositeContainer implements ContainerInterface
      */
     private $containers = [];
 
-    public function get($id)
+    public function get($id, array $parameters = [])
     {
         foreach ($this->containers as $container) {
             try {
+                if (!empty($parameters)) {
```

Comment:
```suggestion
                if ($parameters !== []) {
```

We should use strict comparison since we have a typehint for a parameter. It is faster and cleaner.